common/libs/nodes/rpc/index.js

```diff
@@ -53,15 +57,30 @@ export default class RpcNode extends BaseNode {
           if (item.error) {
             return new Big(0);
           }
-          return new Big(item.result).div(new Big(10).pow(tokens[idx].decimal));
+          return new Big(String(item.result)).div(
+            new Big(10).pow(tokens[idx].decimal)
+          );
         });
       });
+    // TODO - Error handling
   }
 
   async getTransactionCount(address: string): Promise<string> {
     return this.client.call(getTransactionCount(address)).then(response => {
       if (response.error) {
-        throw new Error('getTransactionCount error');
+        throw new Error(response.error.message);
+      }
+      return response.result;
+    });
+  }
+
+  async sendRawTx(signedTx: string): Promise<string> {
+    return this.client.call(sendRawTx(signedTx)).then(response => {
+      if (response.error) {
+        throw new Error(response.error.message);
+      }
+      if (response.errorMessage) {
+        throw new Error(response.errorMessage);
       }
       return response.result;
     });
```

Comment:
Great improvements in here, there was a lot I wasn't sure about regarding the type casting (I assumed making numbers out of big ints would be a problem, but didn't bring it up.) Thanks!